### PR TITLE
Hiding secrets when parsing filestash error messages.

### PR DESCRIPTION
**Pull request by brnovasco**
Hiding secret in filestash related routes.

### CODE REVIEW

File: apps/deepsirius-ui/src/server/api/routers/ssh.ts [hunk 1]
```diff
@@ -179,10 +203,16 @@ export const sshRouter = createTRPCRouter({
...
+        const error = apiErrorSchema.parse(data);
+
+        throw new TRPCError({
+          code: 'INTERNAL_SERVER_ERROR',
+          message: error.message,
+        });
       }
       const buffer = await res.arrayBuffer().catch((err: Error) => {
-        throw new Error(`Failed to fetch buffer file: ${err.message}`);
+        throw new Error(`Failed to fetch buffer file: ${err.message.replace(env.STORAGE_API_KEY, '***')}`);
```
Comment (matyson) [inline apps/deepsirius-ui/src/server/api/routers/ssh.ts]: Not necessary here, the method `.arrayBuffer` does not throw the errors of the external API, i think. Maybe does not even need this catch..

File: apps/deepsirius-ui/src/server/api/routers/ssh.ts [hunk 2]
```diff
@@ -110,16 +117,24 @@ export const sshRouter = createTRPCRouter({
...
+        const data: unknown = await res.json();
+        const error = apiErrorSchema.parse(data);
 
+        throw new TRPCError({
+          code: 'INTERNAL_SERVER_ERROR',
+          message: error.message,
+        });
+      }
       const data = await res.text();
       if (!res.ok) {
```
Comment (matyson) [inline apps/deepsirius-ui/src/server/api/routers/ssh.ts]: Redundant if statement?

**Review by matyson — CHANGES_REQUESTED** [review]

File: apps/deepsirius-ui/src/server/api/routers/ssh.ts (hunk 2 above)
Comment (brnovasco) [reply apps/deepsirius-ui/src/server/api/routers/ssh.ts]: Thanks for catching these unnecessary catches. Should we add this to a patch?

**Review by matyson — COMMENTED** [review]
ok

**Pull request merged**